test/unit/helpers/JsonBuilder.test.ts

```diff
@@ -127,22 +127,18 @@ describe('build json', () => {
             targetRef: 'target_id_0',
           },
         },
-        terminateEventDefinition: {
-          id: 'event_definition_id',
-        },
-        timerEventDefinition: {
-          id: 'event_definition_id',
-        },
+        terminateEventDefinition: { id: 'terminate_event_definition_id_0_1' },
+        timerEventDefinition: { id: 'timer_event_definition_id_2_0' },
         process: [
           {
             id: 'process_participant_0',
             task: {
               id: 'task_id_0_0',
             },
             endEvent: {
-              eventDefinitionRef: 'event_definition_id',
               id: 'event_id_0_1',
               name: 'endEvent',
+              eventDefinitionRef: 'terminate_event_definition_id_0_1',
```

Comment:
**question:** why moving the property? The properties were previously ordered alphabetically, now there is no logical order in addition to an extra change
This remark applies to all places in this file.

Reply:
All these PR are necessary to add unit tests for link. I prefer to cut them by individual change to avoid a big PR where we can't understand what it make.  
  
I order them by what properties is always built and what is not, and I follow the declaration order from the xml of BPMN spec. It's more readable when we want to compare with the specification.  
  
But, I can revert it.

Reply:
> All these PR are necessary to add unit tests for link. I prefer to cut them by individual change to avoid a big PR where we can't understand what it make.

I already know that.
My point was that I don't know where we're going at the end, so I can't give an opinion on implementation without having the full picture (e.g., suggesting alternatives). I'll know more when the source/target link of the event is implemented.

> For this one, if we don"t have a unique id for the LinkEventDefinition, we can't test correctly the source and the target of the events.

IHMO, this deserves to be in the PR description as it explains why the changes are proposed here.

> I order them by what properties is always built and what is not, and I follow the declaration order from the xml of BPMN spec. It's more readable when we want to compare with the specification.

This is not obvious from the test code, and it will probably be difficult to maintain consistency over time without more explicit guidance.

I was just curious to know why the order was changed. There is no need to revert.